src/views/statutory/EditApplication.vue

```diff
@@ -497,11 +497,19 @@ export default {
       // If this is a new application, then allowing selecting bodies which the current user is a member of
       // and not fetching application.
       if (this.isNew) {
-        this.bodies = this.loginUser.bodies
+        for (const body in this.loginUser.bodies) {
```

Comment:
this one can be simplified as

`this.bodies = this.loginUser.bodies.filter(body => this.can.apply_from_body[body.id])`

Reply:
Yes will update it